README.md

```diff
@@ -21,21 +21,34 @@ The Squad Leader template demonstrates best practices for building handheld mili
 
 To run the app:
 
-- Android 4.0.1 or higher
+- Android 4.0.3 or higher
   - Limited functionality when running on an Android emulator ([more info](#notes-on-running-with-the-android-emulator))
 - In the Android device's settings, if Developer Options are available, ensure that **"Don't keep activites"** or **"Do not keep activities"** is **unchecked**. See the [User Guide](documentation/UserGuide.md#uncheck-dont-keep-activities) for details.
 
 To build the app from source:
 
 - Android SDK
-  - Android API 14 or higher
-- [ArcGIS Runtime SDK 10.2.4 for Android](https://developers.arcgis.com/en/downloads/)
-  - Even if you have installed the Eclipse plugin from the online update site, you'll need to download the SDK to get the native binary files (see [build steps](#building-with-eclipse)).
-- Eclipse 3.6.2 or higher (Eclipse for Android recommended)
-- Android Support Library
+  - Android API 15 or higher
+  - Android Support Repository and Android Support Library must be installed, using the Android SDK Manager.
```

Comment:
I initially didn't know what this meant or how to check, but someone more familiar would Android likely would (looks like the Support Library wasn't installed by default, not sure if that explains any of my issues):

![image](https://cloud.githubusercontent.com/assets/3090809/9605830/f37e5eba-508d-11e5-9c30-79fb251e5115.png)